src/test/scala/mesosphere/marathon/integration/setup/MarathonCallbackTestSupport.scala

```diff
@@ -29,29 +33,56 @@ trait MarathonCallbackTestSupport extends ExternalMarathonIntegrationTest {
     marathon.subscribe(callbackUrl)
   }
 
-  override def handleEvent(event: CallbackEvent): Unit = events.add(event)
+  override def handleEvent(event: CallbackEvent): Unit = {
+    log.info(s"Add event to events list: $event")
+    events.add(event)
+  }
 
   def waitForEvent(kind: String, maxWait: FiniteDuration = 60.seconds): CallbackEvent = waitForEventWith(kind, _ => true, maxWait)
 
   def waitForDeploymentId(deploymentId: String, maxWait: FiniteDuration = 30.seconds): CallbackEvent = {
-    waitForEventWith("deployment_success", _.id == deploymentId, maxWait)
+    waitForEventWith("deployment_success", { c: CallbackEvent => c.info("id").toString == deploymentId }, maxWait)
```

Comment:
The `id` method does exactly this, but in one place. I would argue, this is not DRY? Any reason for doing this?